modules/godot/register_types.cpp

```diff
@@ -133,6 +135,18 @@ void register_godot_types() {
 			.add(ECS::register_system(physics_finalize_frame, "physics_finalize_frame")
 							.execute_in(PHASE_MAX, "Physics"));
 
+	// Utilities
+	ECS::register_system_bundle("Utilities")
+			.set_description("Core and global systems: timers, ... (Currently nothing else)")
+			.add(ECS::register_system(timer_updater_system, "TimerUpdaterSystem")
+							.execute_in(PHASE_CONFIG)
+							.set_description("Updates the `TimerDatabag` timestamp"))
```

Comment:
I would just write "Updates the `TimerDatabag`", since the user doesn't know how the TimerDatabag works internally.